scripts/populate_rdev_with_cellguide_data.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+STACK_NAME=""
+SRC_DEPLOYMENT="staging"
+
+function usage {
+    echo "Usage: $0 [-s|--src_deployment <src_deployment>] <stack_name>"
+    echo "Make sure AWS_PROFILE is set to single-cell-dev prior to running this command."
+    echo ""
+    echo "Populates the specified rdev stack with data."
+    echo ""
+    echo "Options:"
+    echo "  -s, --src_deployment    The source deployment to use. Can be either 'staging' or 'dev'. Default is 'staging'."
+    echo ""
+    echo "Arguments:"
+    echo "  stack_name              The name of the rdev stack to be populated."
```

Comment:
Seems unusually long spaces. Is that intentional?

Reply:
Yeah, it's just some formatting
<img width="1008" alt="image" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/16548075/d1a556e3-0277-4628-9256-6c7098a7567b">

Reply:
ah makes sense now thank you!